cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,131 @@
+        })
+    })
+    it('Valida se o campo de texto esta desabilitado ', () => {
+        cy.contains('button', '1').should('be.visible').click()
+        cy.contains('h2', 'Customer Details').should('be.visible')
+        cy.get('input#name').should('be.disabled')
+    });
+
+    it('Valida botão de voltar para lista de clientes ', () => {
+        cy.contains('button', '1').should('be.visible').click()
+        cy.contains('h2', 'Customer Details').should('be.visible')
+        cy.get('input#name').should('be.disabled')
+        cy.contains('button', 'Back').click()
+        cy.contains('h1', 'EngageSphere').should('be.visible')
+    });
+
+    it('Valida links do footer ', () => {
+        cy.contains('p', 'Copyright 2024 - Talking About Testing').should('be.visible')
+        cy.contains('footer a', 'Hotmart').should('have.attr', 'href', 'https://hotmart.com/pt-br/club/talking-about-testing')
+        cy.contains('footer a', 'Udemy').should('have.attr', 'href', 'https://udemy.com/user/walmyr')
+        cy.contains('footer a', 'Blog').should('have.attr', 'href', 'https://talkingabouttesting.com')
+        cy.contains('footer a', 'YouTube').should('have.attr', 'href', 'https://youtube.com/@talkingabouttesting')
+    });
+
+})
+
+describe('Teste de acessibilidade', ()=> {
+    beforeEach(() => {
+        cy.visit('')
+        cy.injectAxe()
+        cy.get('[data-theme="light"]').should('exist')
+    })
+
+    context('Tela inicial', ()=> {
+        it('não encontra problemas de acessibilidade no modo claro', () => {
+            cy.checkA11y()
+        });
+    
+        it('não encontra problemas de acessibilidade no modo escuro', () => {
+            cy.get('#theme-toggle-button').click()
+
+            cy.get('[data-theme="dark"]').should('exist')
+
+            cy.checkA11y()
+        });
+    })
+
+    context('tela de clientes', ()=>{
+        beforeEach(()=>{
+            cy.contains('button', '1').should('be.visible').click()
+
+            cy.contains('h2', 'Customer Details').should('be.visible')
+        })
+        it('não encontra problemas de acessibilidade no modo claro', () => {
+            cy.checkA11y()
+        });
+
+        it('não encontra problemas de acessibilidade no modo escuro', () => {
+            cy.get('button#theme-toggle-button').click()
+        });
+    })
+
+    context('Mostrar endereço', ()=>{
+        beforeEach(()=>{
+            cy.contains('button', '1').should('be.visible').click()
+
+            cy.contains('h2', 'Customer Details').should('be.visible')
+        })
+
+        it('não encontra problemas de acessibilidade no modo claro', () => {
+            cy.get('button.show-address-btn').click()
+
+            cy.checkA11y()
+        });
+
+        it('não encontra problemas de acessibilidade no modo escuro', () => {
+            cy.get('button#theme-toggle-button').click()
+
+            cy.get('button.show-address-btn').click()
+
+            cy.checkA11y()
+        });
+    })
+})
```

Comment:
```suggestion
})

```

É uma convenção (e boa prática) deixar uma linha em branco ao final de cada arquivo.

Dessa forma, ferramentas como o GitHub não exibem o ícone de "No newline at end of file".